x/clerk/client/rest/rest.go

```diff
@@ -0,0 +1,26 @@
+package rest
```

Comment:
we can remove REST entirely

Reply:
Yeah @temaniarpit27 We don't need rest package now since we are using GRPC

Reply:
i was getting errors if package is removed